common.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+let constants = {
```

Comment:
- [ ] Refactor to `init()` method and remove this file

Reply:
Done, please check 69de6b0